<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-50</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="coinInput">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter coin ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="detailsButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Get coin details</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="resultDisplay">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>531</width>
       <height>171</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="priceButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>230</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Get coin Price</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
